plugins/packages/polygon/main.go

```diff
@@ -186,7 +294,82 @@ func Migrate(db *sql.DB, chainid uint64) error {
 		if _, err := db.Exec("UPDATE bor.migrations SET version = 1;"); err != nil {
 			return err
 		}
+		if _, err := db.Exec(`CREATE INDEX bor.logsTxHash ON bor_logs(transactionHash)`); err != nil {
+			log.Error("bor_receiptBlock CREATE INDEX error", "err", err.Error())
+		}
+		if _, err := db.Exec(`CREATE INDEX bor.logsBkHash ON bor_logs(blockHash)`); err != nil {
+			log.Error("bor_receiptBlock CREATE INDEX error", "err", err.Error())
+		}
+	}
+	if schemaVersion < 2 {
+		rows, _ := db.QueryContext(context.Background(), "SELECT parentHash, uncleHash, root, txRoot, receiptRoot, bloom, difficulty, number, gasLimit, gasUsed, `time`, extra, mixDigest, nonce, baseFee from blocks.blocks where coinbase = X'00';")
```

Comment:
While not semantically necessary, it's convention to make SQL keywords like `FROM` and WHERE` uppercase.